### DISCUSSION TITLE
Which service provider's IP can support grass, nodepay, and gradient at the same time

### DISCUSSION THREAD

**Discussion by piqihaha** [engageub/InternetIncome#175] [General]
Which service provider's IP can support grass, nodepay, and gradient at the same time? Do you have any recommendations? The premise is that it should be cheap, preferably no more than 2$ a month.

**Comment by engageub** [comment]
Hello,
Converted this to discussion as it is not an issue. 
A new service provider has been added  where the static residential proxies cost about $1.5 per IP. 
There is also a free trial available as mentioned in the landing page. 
You can sign up and join their discord server to know more before purchasing. 


Link: https://t.co/MBW7AlhT1Z 

Other Providers: https://github.com/engageub/InternetIncome/wiki/Proxies-and-Hosting-Providers

Thank you

**Reply by warhawk200** [reply]
does same "static residential proxies" work with honeygain aswell?

**Reply by engageub** [reply]
Yes. Static Residential Proxies work with honeygain also.

**Reply by warhawk200** [reply]
> Yes. Static Residential Proxies work with honeygain also.

great sir thank you
